doc/DESCRIPTION.md

```diff
@@ -1 +1 @@
-Focalboard is a project management tool that helps define, organize, track and manage work across teams, using a familiar kanban board view.
\ No newline at end of file
+Focalboard is a project management tool that helps define, organize, track and manage work across teams, using a familiar kanban board view. Focalboard comes in two editions: 
```

Comment:
Do not let us hanging, which ones? 😛

Reply:
Well, it is like in the comics, you need to turn the page 😅